netty-server/docker-compose.yml

```diff
@@ -19,24 +19,30 @@ services:
     command: '--data-dir /data --api-key=xyz --enable-cors'
     
   formkiq:
-    image: formkiq/api-server:1.13.0-SNAPSHOT-20231125
+    image: docker.io/formkiq/api-server:latest
     depends_on:
       - minio
       - dynamodb
+      - keycloak
     ports:
       - "8080:8080"
+      # - "8100:8100" #uncomment this line and "ENV JAVA_TOOL_OPTIONS" in Dockerfile to enable debugging
```

Comment:
IMHO local debugging is useful, and it is nice to have possibility to easily re-enable it.